app/models/support/gds/user_facing_components.rb

```diff
@@ -23,6 +23,14 @@ def component_options
             { name: "Specialist Publisher", id: "specialist_publisher" },
             { name: "Content Publisher (beta)", id: "content_publisher" },
             { name: "data.gov.uk", id: "datagovuk" },
+            { name: "Collections Publisher", id: "collections_publisher" },
+            { name: "Contacts Admin", id: "contacts_admin" },
+            { name: "Content Tagger", id: "content_tagger" },
+            { name: "Local Links Manager", id: "local_links_manager" },
+            { name: "Manuals Publisher", id: "manuals_publisher" },
+            { name: "Service Manual Publisher", id: "service_manual_publisher" },
+            { name: "Short URL Manager", id: "short_url_manager" },
+            { name: "Maslow", id: "maslow" },
```

Comment:
Just double checking. Is this a complete list?

Reply:
Looks good, Not sure if we need content data and imminence as well? But maybe we can keep an eye out when we're on support to see if we get any requests for them in the don't know option. 
I'm curious also if there is an order to this list (e.g. most commonly used?) If there isn't might be nice to have them alphabetically.

Reply:
Good shout! I'll add them in! (Ngl i've literally never heard of Imminence haha)

In terms of ordering. I'd assumed the most commonly added applications were at the top. Having said that i think it'd be pretty nice if they were alphabetised. I'll update it to be ordered 👍 

Thanks for the feedback Becca!